grammars/rust.cson

```diff
@@ -322,26 +322,21 @@
   }
   {
     'comment': 'Function call'
-    'match': '\\b([a-zA-Z_][a-zA-Z0-9_]*)\\s*\\('
+    'match': '\\b([a-zA-Z][a-zA-Z0-9_]*)\\s*\\('
     'captures': {
       '1': { 'name': 'entity.name.function.rust' }
     }
   }
   {
-    'comment': 'Function call with type arguments'
-    'begin': '\\b([a-zA-Z_][a-zA-Z0-9_]*)\s*::\s*<'
-    'end': '>\\s*\\('
+    'comment': 'Function call with type parameters'
+    'begin': '\\b([a-zA-Z][a-zA-Z0-9_]*)\\s*(::)(?=\\s*<.+>\\s*\\()'
```

Comment:
`(?=\\s*<.+>\\s*\\()` is needed to correctly tokenize codes like `foo::bar::<i32>()`.